app/views/users/new.html.erb

```diff
@@ -1,52 +1,68 @@
 <% content_for :head do %>
   <%= javascript_include_tag "user" %>
+  <%= javascript_include_tag "auth_providers" %>
 <% end %>
 
-<% content_for :heading_class, "pb-0" %>
-<% content_for :heading do %>
-  <div class='header-illustration new-user-main'>
-    <h1><%= t ".title" %></h1>
+<div class="auth-container">
+  <div class="tabs">
+    <button class="tab-button" onclick="location.pathname='<%= url_for(:action => :new, :controller => :sessions) -%>'"><%= t "sessions.new.tab_title" %></button>
+    <button class="tab-button active" disabled="true"><%= t ".tab_title" %></button>
   </div>
-  <div class='header-illustration new-user-arm d-none d-md-block'></div>
-<% end %>
 
-<div class="row">
-  <div class='text-muted col-sm order-sm-2'>
+  <div class="text-muted col-sm form-container">
```

Comment:
`text-muted` on everything, even on input labels? That's not how they look on images in #4128.

Reply:
Good catch, thanks.